README.md

```diff
@@ -19,14 +19,13 @@ It uses reducers and continuations to create a simple processing pipeline that c
 
 A pipeline consists of following main parts:
 
-- pre-processing functions
-- the main response generating function
-- an optional wrapper function
-- post-processing functions
-- error handling functions
+- pre-processing functions.
+- the main response generating function.
+- an optional wrapper function.
+- post-processing functions.
+- error handling functions.
```

Comment:
You could argue if fullstops here are a bit overkill, but I'll leave that up to you.